packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -304,6 +309,10 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
     }
   }
 
+  public componentWillUnmount(): void {
+    this._events.dispose();
```

Comment:
`BaseComponent` handles this for you already

https://github.com/OfficeDev/office-ui-fabric-react/blob/de80247f560fb65a97d06873a0678f76f7f9744d/packages/utilities/src/BaseComponent.ts#L85

Reply:
I removed this